napari/layers/surface/surface.py

```diff
@@ -81,6 +81,34 @@ class Surface(IntensityVisualizationMixin, Layer):
     cache : bool
         Whether slices of out-of-core datasets should be cached upon retrieval.
         Currently, this only applies to dask arrays.
+    wireframe : bool
```

Comment:
As mentioned above, I think it would be nice to put these all in their own EventedModels - purely for encapsulation reasons. Could do 

```python
layer.wireframe.enabled
layer.wireframe.width
layer.wireframe.color

# and

layer.normals.face.enabled
layer.normals.face.color
layer.normals.face.width
layer.normals.face.length

# and 

layer.normals.vertex.enabled
layer.normals.vertex.color
layer.normals.vertex.width
layer.normals.vertex.length
```

That would probably be my top choice API. What does @alisterburt @kevinyamauchi @jni @alisterburt think?

Reply:
Yeah I agree!